GameOfLife/Calculations/RandomCellGenerator.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GameOfLife
+{
+
+    public class RandomCellGenerator
```

Comment:
You could rename this to CellGenerator/CellPopulator and then take the commented out code with glider creation and add it as a separate method. So you would have a CellGenerator class, that could populate your cells with random values or with specific forms, when needed.